doc/plugins/unittest.rst

```diff
@@ -0,0 +1,69 @@
+###############
+Spyder Unittest
+###############
+
+**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, allowing you to run test suites and view the results in the IDE.
```

Comment:
Add link in unit test to useful docs. Maybe this one: https://docs.python.org/3/library/unittest.html but let me know @ccordoba12 if you have something better.

Reply:
```suggestion
**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, such as Pytest, Unittest and Nose, allowing you to run test suites and view the results in the IDE.
```

- I agree with @juanis2112 that it'd be a nice idea to add links for the testing frameworks we support.
- That's why I added an explicit mention to them above.

Reply:
Links to Pytest, Unittest and Nose was added.